webapp/src/components/Header.tsx

```diff
@@ -35,58 +36,64 @@ export default function Header() {
 	};
 
 	return (
-		<header className="sticky top-0 z-50 w-full flex items-center justify-between px-6 py-4 bg-white border-b border-gray-200">
-			<nav className="flex flex-row">
+		// Header is a flex-col with 2 elements.
+		// Element 1 is a flex-row with the Harbour name on the left and the wallet connection button / info on the right.
+		// Element 2 is the Safe Research banner.
+		<header className="sticky top-0 z-50 w-full flex flex-col items-center px-2 py-2 bg-white border-b border-gray-200">
+			<div className="flex flex-row w-full justify-between mb-2">
 				<Link
 					to="/"
 					className="text-xl font-semibold text-black hover:opacity-75 transition"
 				>
 					Harbour
 				</Link>
-			</nav>
```

Comment:
All these aren't actually changes but just indent changes due to changing how the header is composed. (see code comment)
Is there a way to make this look easier digestable in Github?